modules/ocf_mirrors/manifests/monitoring.pp

```diff
@@ -8,16 +8,18 @@
     $upstream_protocol = 'http',
     $ensure = 'present',
   ) {
-
+  $local_url = "http://mirrors.ocf.berkeley.edu${local_path}/dists/${dist_to_check}/Release"
+  $upstream_url = "${upstream_protocol}://${upstream_host}${upstream_path}/dists/${dist_to_check}/Release"
   if $ensure == 'present' {
     file { "${project_path}/health":
-      content => template("ocf_mirrors/monitoring/${type}-health.erb"),
-      mode    => '0755';
-    }
-
+      ensure => link,
+      target => "/opt/mirrors/bin/${type}-healthcheck",
+      owner  => mirrors,
+      group  => mirrors,
+    }->
```

Comment:
I'd add a space between the `}` and `~` to be more consistent with our other uses of this for chaining